app/models/matched_effort.rb

```diff
@@ -13,4 +13,44 @@ class MatchedEffort < ApplicationRecord
 
   validates_with MatchedEffortOrganisationValidator, if: -> { organisation_id.present? }
   validates_with MatchedEffortCategoryValidator
+
```

Comment:
This is a really nice change, it's much nicer for future maintainability to access this functionality via method calls than by accessing constants and hashtables from another class directly.

Reply:
Thanks! I agnoised over it for a bit, because I wasn't sure if I was overengineering things, but it also helps with building out the form, so useful in two places